aws/policies/BahmniInfraAdmin.json

```diff
@@ -178,6 +178,21 @@
               "route53:ListResourceRecordSets"
           ],
           "Resource": "*"
+      },
+      {
+          "Sid": "EFS",
+          "Effect": "Allow",
+          "Action": [
+              "elasticfilesystem:CreateFileSystem",
+              "elasticfilesystem:DescribeFileSystems",
+              "elasticfilesystem:DeleteFileSystem",
```

Comment:
Delete file system permissions is needed?

Reply:
Yes @mradul1807 . If we want terraform to delete these resources